mobilecoind/src/payments.rs

```diff
@@ -347,6 +347,106 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
         Ok(tx_proposal)
     }
 
+    /// Create and return an SCI that offers to trade one of our inputs for a
+    /// given amount of some currency. This SCI is in the form accepted by
```

Comment:
As mentioned in an another comment, right now the DEQS require partial fills, but we can change that.